lib/config/presets/internal/group.ts

```diff
@@ -208,6 +208,17 @@ const staticGroups = {
     ],
   },
   jsTestMonMajor: {
```

Comment:
Question for the maintainers: we probably want a config migration to migrate users to the correct spelling?

`jsTestMonMajor` -> `jsTestNonMajor`